crates/starknet-os/src/storage/storage_utils.rs

```diff
@@ -68,7 +67,7 @@ pub async fn unpack_blockifier_state_async<S: Storage + Send + Sync, H: HashFunc
 ) -> Result<(SharedState<S, H>, SharedState<S, H>), TreeError> {
     let final_state = {
         let state = blockifier_state.state.clone();
-        state.apply_commitment_state_diff(blockifier_state.to_state_diff()).await?
+        state.apply_commitment_state_diff(blockifier_state.to_state_diff().expect("todo").into()).await?
```

Comment:
`.expect("todo")` seems vague :P

Reply:
Woops, good point.

Reply:
Fixed. This function should not be there btw, this is only used in one integration test. I modified the expect message to be clearer, I'll move the function in another PR.

Reply:
Nice! Thanks!